content/basics/index.md

```diff
@@ -516,10 +472,12 @@ Most of the time this is overkill, but it can be useful:
 ```js
 try {
     somethingBad();
-} catch (e if e instanceof TypeError) {
```

Comment:
This isn't actually in the ES specification:

> Reminder: this functionality is not part of the ECMAScript specification and has been removed in Firefox 59. It's not supported in any current browser anymore.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#Conditional_catch_clauses